public/app/plugins/datasource/loki/datasource.test.ts

```diff
@@ -1349,22 +1356,22 @@ describe('LokiDatasource', () => {
     beforeEach(() => {
       ds = createLokiDatasource(templateSrvStub);
       ds.statsMetadataRequest = jest.fn().mockResolvedValue({ streams: 1, chunks: 1, bytes: 1, entries: 1 });
-      ds.interpolateString = jest.fn().mockImplementation((value: string) => value.replace('$__interval', '1m'));
+      ds.interpolateString = jest.fn().mockImplementation((value: string) => value.replace('$__auto', '1m'));
```

Comment:
This change is unrelated, but we overaly want to default to `$__auto`.

Reply:
👏